### ISSUE TITLE
[Feature] Reject Imported Transactions 

### ISSUE THREAD

**Issue by maxcave** [actualbudget/actual#2953]
### Verified feature request does not already exist?

- [X] I have searched and found no existing issue

### 💻

- [ ] Would you like to implement this feature?

### Pitch: what problem are you trying to solve?

Quite often the Log of transaction in my Bank Account represent a Net payment of multiple transactions. For example: I go out with friends, pay for some things in cash some by card, they pay for some things. And at the end they settle up. 
The transaction which is imported has little meaning as it doesn't represent one flow of spending in one category. Splitting the transaction would also not be ideal I might have spent more in total than the transaction amount.

Long story short: sometimes I would like to be able to record transactions the way they make most sense for me, and ignore the bank transactions. 
I still want to use syncing as in most cases it is correct and convenient. 
I can delete a synced transaction but it comes back when I sync it, I would like to be able to delete it permanently. 
In YNAB (from which I am migrating) this is called rejecting a transaction and is very handy.

I sadly don't have the technical know how to implement it. But I am happy to provide more information if you need any.

Many Thanks
Max

**Comment by youngcw** [comment]
@psybers Did this get added or was it just in discussion?

**Comment by psybers** [comment]
> @psybers Did this get added or was it just in discussion?

@youngcw The ability to delete a transaction and keep it away? As far as I can tell, no. I often utilize that for testing things out - I will delete something and then run the bank sync again to have it re-appear.
